ansible_collections/arista/avd/examples/campus-fabric/intended/configs/SPINE1.cfg

```diff
@@ -0,0 +1,239 @@
+!RANCID-CONTENT-TYPE: arista
+!
+vlan internal order ascending range 1006 1199
+!
+transceiver qsfp default-mode 4x10G
+!
+service routing protocols model multi-agent
+!
+hostname SPINE1
+ip name-server vrf MGMT 8.8.4.4
+ip name-server vrf MGMT 8.8.8.8
+!
+ntp server vrf MGMT pool.ntp.org
+ntp server vrf MGMT time.google.com prefer
+!
+spanning-tree mode mstp
+no spanning-tree vlan-id 4093-4094
+spanning-tree mst 0 priority 4096
+!
+aaa authentication policy local allow-nopassword-remote-login
```

Comment:
not a good/safe practice. I don't think we should have this in the example.

Reply:
will remove